content/70.coi-contribs.md

```diff
@@ -15,4 +15,5 @@
 ### Acknowledgements
 
 We thank Nick DeVito for assistance with the Evidence-Based Medicine Data Lab COVID-19 TrialsTracker data.
+We thank the following contributor for writing (original draft), reviewing and editing of pieces of the text: Yael Evelyn Marshall
```

Comment:
```suggestion
We thank Yael Evelyn Marshall who contributed writing (original draft) as well as reviewing and editing of pieces of the text but who did not formally approve the manuscript.
```
I suggest something like this, which mirrors the deep review language and makes clear that you met the criteria for authorship other than the procedural ones: http://www.icmje.org/recommendations/browse/roles-and-responsibilities/defining-the-role-of-authors-and-contributors.html

Reply:
@cgreene That sounds good, thanks.